similar/main/bm.cpp

```diff
@@ -295,6 +296,7 @@ int gamedata_init()
 
 namespace dsx {
 
+// (currently only ever called if EDITOR is set)
```

Comment:
This is not correct.  This function is included in `!EDITOR` builds and is called during early startup.

```(gdb) bt
#0  d2x::bm_read_all (Vclip=..., fp=0x5555564d5300) at similar/main/bm.cpp:305
#1  0x000055555560c40a in read_hamfile () at common/include/physfsx.h:421
#2  0x000055555560c96e in properties_init () at similar/main/piggy.cpp:1131
#3  0x00005555555922ae in gamedata_init () at similar/main/bm.cpp:288
#4  0x000055555556cbff in d2x::main () at similar/main/inferno.cpp:623
#5  main () at similar/main/inferno.cpp:746
```

Reply:
Ah, indeed. I was probably confused by the fact that it's inside an #if block that according to a comment _used_ to be EDITOR-dependent, but on a second look it was originally `#ifndef EDITOR` and would've been called anyway.

That makes it somewhat more challenging, since I'm currently not sure if the values it sets for `exit_modelnum` and `destroyed_exit_modelnum` matter. It seems that they're initialized to the number of models (i.e. invalid values) on D2 (PIG_VERSION >= 3), which implies they would be loaded later - so only the comment needs to be changed.